client/src/services/loginService.js

```diff
@@ -11,3 +11,11 @@ export async function userLogin(login, password) {
     url: "/api/login",
   });
 }
+
+export async function userLogout() {
+  return await axios({
+    method: "POST",
+    withCredentials: true,
+    url: "/api/logout",
+  });
+}
```

Comment:
should just put this into the logout.js. Better to separate it since this file is called loginService.